crates/model/src/orders/limit_if_touched.rs

```diff
@@ -400,32 +466,28 @@ impl Order for LimitIfTouchedOrder {
         Some(self.is_triggered)
     }
 
-    fn set_position_id(&mut self, position_id: Option<PositionId>) {
-        self.position_id = position_id;
+    // setters that forward into `self`
+    fn set_position_id(&mut self, id: Option<PositionId>) {
```

Comment:
Lets not change the param naming conventions to something this terse.

Reply:
changed, thx